wiki/BBCode/uk.md

```diff
@@ -0,0 +1,320 @@
+# BBCode
+
+**BBCode** це [Мова розмітки](https://uk.wikipedia.org/wiki/Мова_розмітки_даних) яка використовується для спілкування на форумах і, по великому рахунку, більшість форумів в інтернеті. Використовується для урізноманітнення тексту, вона зроблена з тегів для форматування, додавання атрибутів, вкладень, і т.д. Вона використовується повсюди на сайті osu!: в постах на форумах, іменах і описах профілів, і описах мап.
+
+![Редактор публікацій](img/editor.jpg?1 "Редактор публікацій")
+
+## Швидке введення
+
+При натисканні на кнопку форматування без виділення тексту буде створена пара тегів навколо курсора в редакторі. При натисканні на кнопку з виділеним текстом теги будуть створені навколо вибраного тексту.
+
+Користувачі, які хочуть використовувати різні теги разом, можуть накладати їх один поверх іншого, Але, порядок накладання **повинен бути в рамках дозволенного**. При неправильному використанні це може відмінити форматування.  
+
+Правильне і неправильне використання накладених тегів:
+
+- `[centre][b]text[/b][/centre]` Правильно
+- `[b][centre]text[/b][/centre]` Неправильно
+
+## Теги
+
+BBCode, як і багато інших мов розмітки форматує текст використовуючи систему тегів, які починаються і закінчуються квадратними дужками(`[]`). Ці теги розділяються на "відкриваючі" та "закриваючі", які відрізняються додаванням слеша (`/`). Точніше, закриваючі теги містять слеш відразу після відкриваючої дужки, поки відкриваючі теги не містять слеш.
+
+Важливо запам'ятати що відкриваючі теги іноді містять знак дорівнює (`=`) щоб вказати на посилання, розміри шрифту, та інші елементи.
+
+BBCode теги, які використовує сайт детально описані нижче.  
+
+### Жирний шрифт
+
+```
+[b]text[/b]
+```
+
+Тег `[b]` використовується, щоб виділити текст за допомогою жирного шрифту. BBCode виділення не змінює розмір шрифту.
+
+Кнопка на панелі форматування ![Bold button](img/bold.png "Bold")
```

Comment:
seems like you forgot to translate tooltips and alt-texts for those icons (same below)